dags/wb_data.py

```diff
@@ -255,14 +279,28 @@ def load(data: pd.DataFrame):
             rows=data.values.tolist()
         )
 
-    # создаем таблицу
-    create_table()
-    # экспортируем данные
-    data = extract()
-    # преобразовываем данные
-    transformed_data = transform(data)
-    # загружаем данные в целевую таблицу
-    load(transformed_data)
+    send_success_notification = TelegramOperator(
+        task_id='send_success_notification',
+        telegram_conn_id='dag_notification',
+        chat_id='-4239968294',
+        text='DAG prepare_wb_data completed successfully!',
+    )
+
+    send_failure_notification = TelegramOperator(
+        task_id='send_failure_notification',
+        telegram_conn_id='dag_notification',
+        chat_id='-4239968294',
+        text='DAG prepare_wb_data failed!',
+        trigger_rule='one_failed'
+    )
+
+    create_table_task = create_table()
+    extract_task = extract()
+    transform_task = transform(extract_task)
+    load_task = load(transform_task)
+
+    create_table_task >> extract_task >> transform_task >> load_task >> send_success_notification
+    create_table_task >> extract_task >> transform_task >> load_task >> send_failure_notification
```

Comment:
Можно так:
```suggestion
    create_table_task >> extract_task >> transform_task >> load_task >> [send_failure_notification, send_success_notification]
```
Или так:
```suggestion
    chain(create_table_task, extract_task, transform_task, load_task, [send_failure_notification, send_success_notification])
```